WebDriverAgentLib/Utilities/FBImageUtils.m

```diff
@@ -66,22 +67,32 @@ BOOL FBIsPngImage(NSData *imageData)
 #else
 NSData *FBAdjustScreenshotOrientationForApplication(NSData *screenshotData, UIInterfaceOrientation orientation)
 {
+
   UIImageOrientation imageOrientation;
-  if (SYSTEM_VERSION_LESS_THAN(@"11.0")) {
-    // In iOS < 11.0 screenshots are already adjusted properly
-    imageOrientation = UIImageOrientationUp;
-  } else if (orientation == UIInterfaceOrientationLandscapeRight) {
-    imageOrientation = UIImageOrientationLeft;
-  } else if (orientation == UIInterfaceOrientationLandscapeLeft) {
-    imageOrientation = UIImageOrientationRight;
-  } else if (orientation == UIInterfaceOrientationPortraitUpsideDown) {
-    imageOrientation = UIImageOrientationDown;
+  if (FBConfiguration.screenshotOrientation == UIInterfaceOrientationUnknown) {
+    if (orientation == UIInterfaceOrientationLandscapeRight) {
```

Comment:
` if (SYSTEM_VERSION_LESS_THAN(@"11.0")) {` is missing here. The previous implementation should be kept unchanged